db/migrate/20221205094552_create_github_team_members.rb

```diff
@@ -0,0 +1,12 @@
+class CreateGithubTeamMembers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :github_team_members do |t|
+      t.string :username, null: false
+      t.string :team, null: false      
```

Comment:
I opted to just store the team name in this table and not create another `Github::Team` table